src/sentry/api/endpoints/project_plugin_details.py

```diff
@@ -87,6 +90,20 @@ def post(self, request: Request, project, plugin_id) -> Response:
         if not plugin.can_disable:
             return Response({"detail": ERR_ALWAYS_ENABLED}, status=400)
 
+        for feature_description in plugin.feature_descriptions:
+            if feature_description.featureGate == IntegrationFeatures.SESSION_REPLAY:
```

Comment:
@oioki i don't think we need this, can we just check the feature flag for data forwarding?

Reply:
I'd like to keep some sort of safe logic in case if we add a new plugin. How likely it is?
I can change this logic to only check for data forwarding, but what if we add a new plugin with new feature and forget about adding the check here. Maybe we can add a comment in some other place where this requirement will be salient?

Reply:
@oioki 0% chance we add a new plugin, it's legacy system that we want to get rid of